src/views/Balance.vue

```diff
@@ -1,30 +1,225 @@
 <template>
     <h2 class="text-gray-400 text-3xl font-medium">Balance</h2>
     <p class="text-gray-400">Account: {{ allAccounts ? allAccounts[0] : 'no account selected' }}</p>
-    <!-- <p class="text-gray-400">Balance: {{ balance ? balance.toString() : '0' }}</p> -->
+    <p class="text-gray-400">Balance: {{ balance ? balance.toString(10) : 'cannot read' }}</p>
     <p class="text-gray-400">Counter: {{ testCounter ? testCounter.toString() : '0' }}</p>
 
     <p class="text-gray-400">{{ api ? api.isConnected : 'no api' }}</p>
+    <div class="flex flex-col flex-1 overflow-y-auto lg:overflow-hidden">
+        <!-- <%- include('./components/common/_header-mobile'); -%> -->
+
+        <main class="flex-1 relative z-0 lg:overflow-y-auto overflow-x-hidden focus:outline-none">
```

Comment:
I think it's better to move this to the `DApps.vue` page instead of using it in the Balance page.